wrappers/csharp/Intel.RealSense/Sensors/SoftwareSensor.cs

```diff
@@ -19,7 +19,7 @@ public void AddVideoFrame(SoftwareVideoFrame f)
             NativeMethods.rs2_software_sensor_on_video_frame(Handle, f, out error);
         }
 
-        public void AddVideoFrame<T>(T[] pixels, int stride, int bpp, double timestamp, TimestampDomain domain, int frameNumber, VideoStreamProfile profile)
+        public void AddVideoFrame<T>(T[] pixels, int stride, int bpp, double timestamp, TimestampDomain domain, int frameNumber, VideoStreamProfile profile, float depthUnits = 0.001f)
```

Comment:
I think the default should be 0, not 0.001.
That way, it will check `DEPTH_UNITS` after I do my own PR.
We cannot assume to know what units the user uses.

Reply:
Done. Default to 0.
Also added and initialized the depth unit when constructing sw depth frame in the software-device tutorial.